### PR TITLE
Create Account

### PR DESCRIPTION
**Pull request by ChihChengLiang**

### CODE REVIEW

File: contracts/CreateAccount.sol [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        bytes32 newRoot = UpdateAccountWithSiblings(
+            createdAccount,
+            accountProofs.to // We only use the pathToAccount and siblings but not the proof itself
+        );
+        bytes memory createdAccountBytes = RollupUtils.BytesFromAccount(
+            createdAccount
+        );
+
+        return (newRoot, createdAccountBytes, "", 0, true);
+    }
```
Comment (ChihChengLiang) [inline contracts/CreateAccount.sol:152]: The core of the create Account.

File: contracts/CreateAccount.sol [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            bytes32,
+            bytes memory,
+            bytes memory,
+            uint256,
+            bool
+        )
+    {
+        Types.UserAccount memory createdAccount;
+        createdAccount.ID = _tx.toIndex;
+        createdAccount.tokenType = 1; // Arbitrary assign a default token
```
Comment (barryWhiteHat) [inline contracts/CreateAccount.sol:129]: Would be nice ot have this in the commimtnet.

Comment (ChihChengLiang) [reply contracts/CreateAccount.sol:129]: Added 👍

**Comment by ChihChengLiang** [comment]
Post PR comments welcomed

**Pull request merged**